classes/CronJobsForms.php

```diff
@@ -320,10 +320,20 @@ public static function getTasksListValues()
                 $cron['task'] = urldecode($cron['task']);
             }
 
+            $weekdays = array(
```

Comment:
Eveyrthing is already generated here: https://github.com/PrestaShop/cronjobs/blob/ca3575f0a30a52f70368cb4e484bae2c8d3f4f61/classes/CronJobsForms.php#L378